src/home.js

```diff
@@ -1,36 +1,63 @@
-
-const displayPizza = document.querySelector('.display-pizza');
+import { getLikes } from './hitApi.js';
 
 
-const showPizza = (value) => {
+const displayPizza = document.querySelector('.display-pizza');
+const showPizza = async (value) => {
   value.forEach((dat) => {
     const divContainer = document.createElement('div');
     divContainer.className = 'pizza-container';
     const image = document.createElement('img');
     image.alt = 'pizza-image';
     image.src = dat.image_url;
 
+    const likes = document.createElement('span');
+    likes.innerHTML = '\u2661';
+    likes.className = 'like-me';
+
     const namePub = document.createElement('span');
     namePub.innerHTML = dat.publisher;
 
     const namePizz = document.createElement('span');
-    namePizz.innerHTML = dat.title;
+    namePizz.innerHTML = '5 likes';
+    
+
+    namePizz.className = 'likes-span';
 
     const commentBtn = document.createElement('button');
     commentBtn.innerHTML = 'Comment';
 
     const reservationBtn = document.createElement('button');
     reservationBtn.innerHTML = 'Reservation';
 
+    likes.addEventListener('click', async () => {
+      await fetch(
+        'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/8FcrK9POw5EbfAJUs4DD/likes',
```

Comment:
this link, which hits the Api, you used it more than once, so to respect the Dry principle, please store it in a variable.

Reply:
Hi @nicupop729 ,

All changes will be duly attended to as requested.

Thank you for the feedback.